src/layers/core/path-layer/path-layer-vertex-64.glsl.js

```diff
@@ -187,14 +194,38 @@ void main() {
   project_position_fp64(currPosition64, projected_curr_position);
   float projected_curr_position_z = project_scale(currPosition.z);
 
-  currPosition = project_position(currPosition);
+  // Calculate previous position
+
+  vec3 prevPosition = mix(-instanceLeftDeltas, vec3(0.0), isEnd);
+
+  // Calculate prev position 64bit
+
+  vec2 tempPrevPosition64[2];
+  tempPrevPosition64[0] = sum_fp64(vec2(prevPosition.x, 0.0), instanceStartPositions64[0]);
+  tempPrevPosition64[1] = sum_fp64(vec2(prevPosition.y, 0.0), instanceStartPositions64[1]);
+
+  vec4 prevPosition64 = vec4(tempPrevPosition64[0].xy, tempPrevPosition64[1].xy);
+
+  vec2 projected_prev_position[2];
+  project_position_fp64(prevPosition64, projected_prev_position);
```

Comment:
This line looks like it could cost us some performance.

Do we still have the render benchmark support? Could we collect before and after numbers?